src/app/modules/ix-forms/components/ix-input/ix-input.component.html

```diff
@@ -39,11 +39,17 @@
     <i class="material-icons" matTooltip="Show" *ngIf="!showPassword">visibility_off</i>
   </button>
 </div>
+
+<div *ngIf="linkText">
```

Comment:
Eh, why have another property? Why not just add this below the input in component template?

Reply:
In #6526 I used `[hint]` to show additional information below the input. Maybe it can be changed to render html.

Reply:
My initial thought process was that because of the margins and everything that IxInputs have, a new element under the input would look out of place. This put it right next to the input.

Reply:
In ix-input, it's possible to adjust margins in parent component.